src/commands/appSettings/connectionSettings/eventHubs/IEventHubsConnectionWizardContext.ts

```diff
@@ -3,17 +3,17 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { ResourceManagementModels } from "@azure/arm-resources";
-import { ISubscriptionContext } from "@microsoft/vscode-azext-utils";
+import type { ResourceGroup } from "@azure/arm-resources";
 import { EventHubsConnectionTypeValues, StorageConnectionTypeValues } from "../../../../constants";
+import { IEventHubWizardContext } from "../../../addBinding/settingSteps/eventHub/IEventHubWizardContext";
 import { ISetConnectionSettingContext } from "../ISetConnectionSettingContext";
 
-export interface IEventHubsConnectionWizardContext extends ISetConnectionSettingContext, Partial<ISubscriptionContext> {
-    resourceGroup?: ResourceManagementModels.ResourceGroup;
+export interface IEventHubsConnectionWizardContext extends IEventHubWizardContext, ISetConnectionSettingContext {
+    resourceGroup?: ResourceGroup;
 
     // Connection Types
     azureWebJobsStorageType?: StorageConnectionTypeValues;
     eventHubsConnectionType?: EventHubsConnectionTypeValues;
 
     // Netherite uses all of the eventhub namespace settings in IEventHubWizardContext
-}
+}``
```

Comment:
oopsies

Reply:
Damn it. The theme I _was_ using had quotes be like dark gray on a dark gray background so they were impossible to see.

Though I got rid of them and pushed those changes. Not sure why it's still showing up here

![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/d943740c-1382-4efd-913d-ef332a4d6dc0)